### PR TITLE
Data Browser Overview

### PR DESCRIPTION
**Pull request by ojbravo**
UI Update to match Homepage

### CODE REVIEW

File: src/data-browser/ImageCard.scss [hunk 1]
```diff
@@ -1,4 +1,116 @@
...
+
+  .card-wrapper {
+    margin-bottom: 1em;
+    width: 100%;
+
+    .ImageCard {
+      display: flex;
+      flex-direction: column;
+      background-color: var(--hmda-blue);
+      //background-color: #eeeeee;
```
Comment (meissadia) [inline src/data-browser/ImageCard.scss:25]:
```suggestion
```

File: src/data-browser/ImageCard.scss [hunk 2]
```diff
@@ -1,4 +1,116 @@
...
+
+
+
+
+
+
+
+/*a.card {
   text-decoration: none;
 }
```
Comment (meissadia) [inline src/data-browser/ImageCard.scss:116]:
```suggestion
```

File: src/data-browser/ImageCard.scss [hunk 3]
```diff
@@ -153,4 +265,4 @@ a:focus .ImageCard.enabled .desc {
   .DataBrowser .home .card-wrapper {
     flex: 1 0 30%;
   }
```
Comment (meissadia) [inline src/data-browser/ImageCard.scss:267]:
```suggestion
```

**Review by meissadia — APPROVED** [review]
Some dead code that could be removed but testing in DEV environment looks good 👍🏾

File: src/data-browser/ImageCard.scss (hunk 1 above)
Comment (ojbravo) [reply src/data-browser/ImageCard.scss:25]: Done.

File: src/data-browser/ImageCard.scss (hunk 2 above)
Comment (ojbravo) [reply src/data-browser/ImageCard.scss:116]: Removed.

**Pull request merged**